Adyen/UI/Form/Items/Label/FormLabelItemView.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2020 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// A view representing a header item.
+internal final class FormLabelItemView: FormItemView<UILabel> {
```

Comment:
Treating the `UILabel` as both a model object (`FormItem`) and a view (`AnyFormItemView`) makes me cringe a bit :-D , also It can be very confusing two years from now :-) .